recipes/abyss/1.5.2/create-wrapper.sh

```diff
@@ -14,6 +14,6 @@ chmod a-x "$2"
 
 cat >"$1" <<EOF
 #!/bin/bash
-exec $(sed 's|^#!/\(usr/\)\?bin/||;q' "$2") '$2' "\$@"
+exec $(sed -E 's|^#!(/\usr)?\/bin/||;q' "$2") '$2' "\$@"
```

Comment:
There is a typo `/\`, but we don't have to escape the `/` anyway since we use `|` as the delimiter.
Maybe we better make this
```bash
interpreter="$(sed -nE '1 s|^#! *(/usr)?/bin/||p;q' "$2")"
[ -n "${interpreter}" ]
cat >"$1" <<EOF
#!/bin/bash
exec ${interpreter} '$2' "\$@"
EOF
```
to catch that failure already at build time.

Reply:
Or even `exec '${PREFIX}/'${interpreter} '$2' "\$@"` for the `exec` part.

Reply:
Is `${PREFIX}` necessary, given the path and `/usr/bin/env ${interpreter}` should be working correctly? I'm not sure which is better here; implicit or explicit? I do like defining the interpreter on its own line though.

Reply:
Also, one _possible_ reason not to use `-E` extended is that I saw somewhere it's slower. Since we're literally only searching 1 line I'm sure it's not even measureable but would conceivably be a difference.

Reply:
And we are calling that `sed -E` only once at *build time* :wink:.
There are pros and cons for using the full path or not. Generally, I think we should prefer absolute paths. Often times we can't due to those massive shebang restrictions -- esp. on Linux, see https://www.in-ulm.de/~mascheck/various/shebang/#results (shebangs were just not engineered for today's relocatable environments...). As is, we're not consistent in that regard in Bioconda overall, so it's not that important, I guess. Currently, either way should be fine, feel free to decide.